pebble/src/main/java/com/mitchellbosecke/pebble/extension/core/DateFilter.java

```diff
@@ -34,6 +42,7 @@ public DateFilter() {
     this.argumentNames.add("format");
     this.argumentNames.add("existingFormat");
     this.argumentNames.add("timeZone");
+    this.argumentNames.add("alterTime");
```

Comment:
the format will be `P0DT1H:1M:1S`, but can be shortened to `P0DT1H` or `P1D`.

Would it be worth introducing longer time frames as well? e.g. years, months?

Reply:
We could potentially add two fields. `alterTime` and `alterDate`.